plugin.php

```diff
@@ -27,7 +27,13 @@ function classloader($class) {
     if ($ns_offset === NULL) {
       $ns_offset = strlen(__NAMESPACE__) + 1;
     }
-    include __DIR__ . '/src/' . strtr(substr($class, $ns_offset), '\\', '/') . '.php';
+    $folder = '/src/';
+    $class_name = strtr(substr($class, $ns_offset), '\\', '/');
+    if (strpos($class_name, 'Widgets/') === 0) {
+      $folder .= 'widgets/';
+      $class_name = substr($class_name, 8);
+    }
+    include __DIR__ . $folder . $class_name . '.php';
```

Comment:
Please revert this and change the folder name (and thus sub-namespace name) from `widgets` to `Widgets`.